repos/spack_repo/builtin/packages/paraview/paraview-cdireader-lazy.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/Documentation/release/dev/cdireader-lazy.md b/Documentation/release/dev/cdireader-lazy.md
```

Comment:
We should be able to drop the release note part of the patch, the only part that is needed is setting the read mode to lazy.

Reply:
Yeah, I thought about that, but I choose to not "diverge" from the PR.

I agree with you anyway, I'll do it.

Reply:
see https://github.com/spack/spack-packages/pull/2671/commits/0d748f972ee472d26d7a79ab4521655901c3ee15